go.mod

```
@@ -1,6 +1,8 @@
 module k8s.io/release
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.3
```

can we bump the go mod 

```suggestion
go 1.23.4 
```

---

Yes, sure :+1: